server/config.ts

```diff
@@ -250,6 +250,7 @@ export const auth = {
     host: envVar({
       name: "MODIACONTEXTHOLDER_HOST",
     }),
+    removePathPrefix: true,
```

Comment:
Usikker på om denne egentlig må settes til `true`. Fungerer fint uten å fjerne `modiacontextholder` fra kallet i `syfooversikt`.
Mulig jeg misforstår hva det blir snakket om [her](https://nav-it.slack.com/archives/CLJJKQEHY/p1719570994556859?thread_ts=1719561658.366889&cid=CLJJKQEHY) 🤔

Reply:
Kanskje forsøke uten denne i første omgang? (dette vil fungere likt i dev-gcp og prod-gcp, så finner fort ut om det er noe som trengs)